docs/BreakingChanges/168 DataContractJsonSerializer incompatible with ES6 (and V8).md

```diff
@@ -0,0 +1,35 @@
+## 168: DataContractJsonSerializer incompatible with ES6 (and V8)
+
+### Scope
+Edge
+
+### Version Introduced
+4.7
+
+### Source Analyzer Status
+Planned
+
+### Change Description
+Starting in .NET Framework .NET 4.7, the compatibility issue between the escaped strings for some special control characters generated by DataContractJsonSerializer and ECMAScript V6 has been fixed. 
```

Comment:
I'd change this to:
In the .NET Framework 4.6.2 and earlier versions, the DataContractJsonSerializer did not serialize some special control characters, such as `\b`, `\f`, and `\t`, in a way that was compatible with the ECMAScript V6 and V8 standards. Starting with the .NET Framework 4.7, serialization of these control characters is compatible with ECMAScript V6 and V8.